fern/pages/src/mcp/providers.mdx

```diff
@@ -122,22 +136,42 @@ const composio = new Composio({
   provider: new MastraProvider(),
 });
 
-// Create an MCP server with the Gmail toolkit
-const mcpConfig = await composio.mcp.create(
-"Gmail Readonly Server",
-[
-  {
-    toolkit: "GMAIL",
-    allowedTools: ["GMAIL_FETCH_EMAILS"],
-    authConfigId: "<auth_config_id>"
-  },
-],
-{ useComposioManagedAuth: true }
-);
+// Create an MCP server with the Gmail toolkit using new API structure
+const mcpConfig = await composio.mcp.create({
+  name: "Gmail Readonly Server",
+  serverConfig: [
+    {
+      authConfigId: "<auth_config_id>", // Your auth config ID
+      allowedTools: ["GMAIL_FETCH_EMAILS"]
+    }
+  ],
+  options: {
+    isChatAuth: true
+  }
+});
 
 console.log(`✅ MCP server created: ${mcpConfig.id}`);
 console.log(`🔧 Available toolkits: ${mcpConfig.toolkits.join(", ")}`);
 
+// Check user connection status
+const connectionStatus = await composio.mcp.getUserConnectionStatus({
+  id: mcpConfig.id,
+  userId: "utkarsh-dixit"
+});
+
+console.log("Connection status:", connectionStatus.connected);
+
+if (!connectionStatus.connected) {
```

Comment:
In the above example, I can see `isChatAuth` is true, so the auth can happen in the client as well. 
We don't need to show this console here. 

Or lets mark `isChatAuth` as false so that the below console statement related to it. makes sense?